packages/babel-cli/src/babel/dir.js

```diff
@@ -43,18 +43,24 @@ module.exports = function (commander, filenames) {
     }
   }
 
+  let filterDotFiles = function (x) {
+    return x[0] === ".";
+  };
+  let copyFilesFilter = () => false;
+
   function handle(filename) {
     if (!fs.existsSync(filename)) return;
 
     let stat = fs.statSync(filename);
-
     if (stat.isDirectory(filename)) {
       let dirname = filename;
-
-      _.each(util.readdir(dirname), function (filename) {
+      if (commander.copyDotFiles) {
+        copyFilesFilter = filterDotFiles;
```

Comment:
Oh, I thought it'll be along with `--copy-files` as per the [discussion](https://github.com/babel/babel/issues/5052#issuecomment-270629590).